File: RagnarokBotWeb/ClientApp/src/app/models/account.dto.ts
import { ScumServer, ScumServerResponse } from "./scum-server";

export class AccountDto {
    name!: string;
    lastName!: string;
    email!: string;
    serverId!: number;
    server?: ScumServer;
    servers?: ScumServer[];
    accessLevel!: number;
}

File: RagnarokBotWeb/Domain/Services/Dto/AccountDto.cs
using RagnarokBotWeb.Domain.Enums;

namespace RagnarokBotWeb.Domain.Services.Dto
{
    public class AccountDto
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public string LastName { get; set; }
        public long ServerId { get; set; }
        public string? Country { get; set; }
        public IEnumerable<ScumServerDto> Servers { get; set; }
        public ScumServerDto? Server { get; set; }
        public AccessLevel AccessLevel { get; set; }
    }
}
